test/views/CrystallisedViewSpec.scala

```diff
@@ -307,6 +307,20 @@ class CrystallisedViewSpec extends TestSupport {
       document.getElementById("changes").text shouldBe crysMessages.changes
     }
 
+    "NOT show a button to go to payments, when the payment feature is disabled" in {
+      mockAppConfig.features.paymentEnabled(false)
+      val setup = pageSetup(busPropBRTCalcDataModel, testIncomeSources)
+      import setup._
+      document.getElementById("payment-button") should be(null)
+    }
+
+    "show a button to go to payments, when the payment feature is enabled" in {
```

Comment:
Can you add some tests to prove that:
- The wording on the button is correct
- The link on the button goes to the correct location